homu/server.py

```diff
@@ -29,6 +29,10 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
+    logger = g.logger.getChild('queue')
+
+    logger.debug('repo_label: %s', repo_label)
```

Comment:
I ended up changing it since the solution to https://github.com/barosl/homu/pull/74#discussion-diff-26766972 works well for this too.